src/languageServerInstaller.ts

```diff
@@ -10,6 +10,13 @@ import * as yauzl from 'yauzl';
 
 const releasesUrl = "https://releases.hashicorp.com/terraform-ls";
 
+interface Release {
```

Comment:
this type is also referenced in the `installPkg` method, you should probably update that as well

Reply:
Good catch - fixed.